### PR TITLE
Update Flutter_map on pubspec.yaml

### PR DESCRIPTION
**Pull request by X-SLAYER**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
     sdk: flutter
   pedantic: ^1.8.0
   extra_pedantic: ^1.1.1+3
-  flutter_map: ^0.8.2
+  flutter_map: any
```
Comment (X-SLAYER) [inline pubspec.yaml]: or use this to prevent any new updates in future 
flutter_map: ">=0.8.0 <1.0.0"

**Comment by synw** [comment]
Sorry but `any` is not acceptable when you publish a package on pub.dev. It has to be a version number

**Comment by X-SLAYER** [comment]
try with this
flutter_map: ">=0.8.0 <1.0.0"

**Comment by X-SLAYER** [comment]
can you update it this to the last flutter_map version !

**Pull request closed without merging**